scripts/1.1.3_preupgrade.sh

```
@@ -0,0 +1,98 @@
+  local dest="$2"
+  local desc="$3"
+  [ -f "$src" ] && cp "$src" "$dest" && log "Successfully replaced $desc" || error_exit "$src does not exist. Aborting."
+}
+
+update_chain_id() {
+  local genesis="$1"
+  local client_toml="$2"
+  local chain_id
+  chain_id=$(jq -r '.chain_id' "$genesis")
+  # check if chain_id contain special characters like / &
+  if [[ "$chain_id" =~ [\/&] ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)